workflow-report/action.yml

```diff
@@ -0,0 +1,77 @@
+name: 'Sends VK Teams notifications about failed job'
+description: 'This action creates a message about failed job and sends it to VK Teams chat'
+
+inputs:
+  api-url:
+    description: 'Bot API URL (ICQ/Myteam)'
+    required: false
+    default: 'https://api.internal.myteam.mail.ru/bot/v1/'
+  bot-token:
+    description: 'Bot token'
+    required: true
+  chat-id:
+    description: 'Notification chat ID (or stamp from chat URL)'
+    required: true
+  job-steps:
+    description: 'Get steps to show it in message'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Compile message about workflow failure
```

Comment:
workflow -> job

Reply:
Fixed, searched in text of readme and action - should be no more "workflow" (except "# Prepare your workflow" and "uses: tarantool/actions/workflow-report@master")